apps/coscrad-frontend/src/components/nav-bar/nav-bar.tsx

```diff
@@ -45,6 +45,10 @@ export const NavBar = (): JSX.Element => {
             link: routes.treeOfKnowledge,
             label: 'Tree of Knowledge',
         },
+        {
```

Comment:
This needs to be a dynamic link. We want the `listenLive` property on the content config to be optional. If it's not provided, we don't render the route.  Not all groups have a radio.

Reply:
done